apps/dash-lunar-lander/app.py

```diff
@@ -0,0 +1,428 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import numpy as np
+import casadi
+from casadi import SX, DM
+from math import cos, sin
+import plotly.graph_objects as go
+
+external_stylesheets = ["https://codepen.io/chriddyp/pen/bWLwgP.css"]
```

Comment:
Let's just avoid loading any scripts from remote URL, users might download this app and want to be able to run it locally offline. 
Simply copy this stylesheet in a `base.css` file and add into assets/ folder.